Dockerfile.upload_success

```diff
@@ -3,7 +3,8 @@ FROM public.ecr.aws/lambda/python:3.8
 COPY backend/layers/processing/upload_success .
 COPY backend/layers ./backend/layers
 
-RUN pip3 install -r requirements.txt
+ADD requirements-upload-success.txt equirements-upload-success.txt
+RUN pip3 install -r equirements-upload-success.txt
```

Comment:
```suggestion
ADD requirements-upload-success.txt requirements-upload-success.txt
RUN pip3 install -r requirements-upload-success.txt
```
Tested?

Reply:
@atolopko-czi - I realized I didn't test it properly locally. I added a mods to the `.dockerignore` and tested docker container creation locally! Thanks for catching this.